tools/AutoTuner/src/autotuner/distributed.py

```diff
@@ -882,6 +882,7 @@ def sweep():
     print('[INFO TUN-0009] Waiting for results.')
     ray.get(workers)
     print('[INFO TUN-0010] Sweep complete.')
+    sys.exit(0)
```

Comment:
This should not be needed as the script will end after the call of `sweep()`. We should only exit with an error in places other than `__main__.`